### DISCUSSION TITLE
Allowing URLs in the text

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1424] [Ideas]
I'm sorry if this idea has already been posted; I couldn't find it.

The basic idea would be a special tag for URLs in the text (such as notes) to refer websites (wikipedia for example). A quick click on the url would open the website in the system's default browser.

For a story I'm writing I need to reference the Kardashev scale (and related pages) on wikipedia. It would be helpful if I could simply drop a link in a note or, better yet, create a link in the project tree.

**Comment by vkbo** [comment]
It has been discussed before, and is implied in the sticky issue #815. Basically, novelWriter is not designed for research notes. The general recommendation is to use one of the many tools designed for this specific purpose.

The main reason for this limitation is simply scope. I do most of the development myself, and sticking to a narrow scope for writing fiction makes the project more maintainable. Especially since the editor is plain text. That said, allowing for standard Markdown URLs isn't a very complicated feature, so feel free to open a feature request on it and I'll look at it,

Another feature that has been discussed is an attachement manager for notes. This may be a more universal solution where multiple kinds of references can be added to notes without having to be implemented as a part of the markup syntax. It would be straightforward to make one of them an external URL. Handling URLs in novelWriter is itself trivial. It's already done for documentation and I plan to add a web dictionary integration as well.
